content/fr/zero-trust-architecture.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Architecture Zéro Confiance
+status: Completed
+category: concepts
+tags: ["sécurité", "", ""]
+---
+
+## Ce que c'est
+
+L'architecture zéro confiance est une approche de conception et de mise en place des systèmes informatiques qui élimine complètement la notion de confiance. 
+Son principe fondamental repose sur le credo "ne jamais faire confiance, toujours vérifier". 
```

Comment:
Il manque la traduction de la partie `,devices or systems themselves, whilst communicating to other components of a system, always verify themselves before doing so.`
Je propose `"ne jamais faire confiance, toujours vérifier", les dispositifs ou systèmes eux-mêmes, lorsqu'ils communiquent avec d'autres composants d'un système, se vérifient toujours avant de communiquer.`

Reply:
Ops j'ai sauté un morceau de la phrase😅